blockstore/apps/api/v1/tests/test_views.py

```diff
@@ -139,28 +177,45 @@ def test_list(self):
 
     def test_get(self):
 
-        url = reverse('api:v1:bundleversion-detail', kwargs={
-            'bundle_uuid': self.bundle_version.bundle.uuid, 'version_num': self.bundle_version.version_num
+        response = self.response('api:v1:bundleversion-detail', kwargs={
+            'bundle_uuid': self.bundle_version.bundle.uuid,
+            'version_num': self.bundle_version.version_num,
         })
-        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
If you modify `self.response()` to have an optional `expected_response_code=200` parameter and put the `self.assertEqual(response.status_code, expected_response_code)` line inside that method, you can remove the repeated `self.assertEqual(response.status_code, 200)` from all these tests.

Reply:
Nice, thanks -- addressed with 206984e .